mysql-test/t/percona_slow_query_log_use_global_control.test

```diff
@@ -25,6 +25,23 @@ SET SESSION query_exec_time=default;
 
 --source include/log_stop.inc
 
+--echo #
+--echo # Bug PS-5813: Setting 'none' value for slow_query_log_use_global_control throwing error.
```

Comment:
Can't this go into `mysql-test/suite/sys_vars/t/slow_query_log_use_global_control_basic.test`? That's where basic value validation tests are usually placed.

Reply:
Done